softdevice_controller/CHANGELOG.rst

```diff
@@ -46,14 +46,17 @@ Bugfixes
 Known issues and limitations
 ============================
 
-* The RSSI value reported by the Softdevice Controller is the raw value from the radio peripheral.
-  Some SoCs require compensation of the RSSI value based on the chip temperature.
-  See the Errata document for the respective SoC for detailed information.
+The RSSI value reported by the Softdevice Controller is the raw value from the radio peripheral.
+Some SoCs require compensation of the RSSI value based on the chip temperature.
+See the Errata document for the respective SoC for detailed information.
+
+For the nRF53 Series, you can retrieve the chip temperature by reading the value of the temperature peripheral on the network core.
+To do this with the SoftDevice Controller, use the Zephyr HCI VS Read Chip Temperature command (``BT_HCI_OP_VS_READ_CHIP_TEMP``).
+  
+For the nRF52 Series, you can use the Zephyr sensor API instead of the HCI command to retrieve the chip temperature.
+You can then use the retrieved temperature value to compensate the raw RSSI value, following the workaround in the Errata document.
```

Comment:
These lines would look better as two bullet points, I think.

Reply:
These paragraphs are misleading (and bullet points would be even worse), since we're giving two ways of retrieving the temperature, and then the last sentence tells you what you do with the temperature.

Reply:
Please either combine into one paragraph or split out the last sentence into another paragraph.
Line 53/54 state how to get the temperature value on nRF53, line 56 states how to get it on nRF52, and line 57 states what to do with the retrieved value.